src/Components/Patient/UpdateStatusDialog.tsx

```diff
@@ -40,9 +36,9 @@ const styles = {
   },
 };
 
-interface URLS {
-  [id: string]: string;
-}
+// interface URLS {
+//   [id: string]: string;
+// }
```

Comment:
_If this is an unused interface or variable, removing them would be better instead of commenting them out_